avm/res/resources/deployment-script/main.bicep

```diff
@@ -0,0 +1,256 @@
+metadata name = 'Deployment Scripts'
+metadata description = 'This module deploys Deployment Scripts.'
+metadata owner = 'Azure/module-maintainers'
+
+// ================ //
```

Comment:
Please move the definitions to the end of the file so that e.g. the inputs come first.